### DISCUSSION TITLE
Expand All is not working

### DISCUSSION THREAD

**Discussion by aidantorrence** [community/community#46410] [Pull Requests]
### Select Topic Area

Bug

### Body

<img width="949" alt="image" src="https://user-images.githubusercontent.com/26528573/216807267-449cd0cd-bae4-46fc-9bd5-cbda8ce58270.png">

This button is not working, am I crazy?  I've tried on multiple laptops.

**Comment by NHbinaaa111** [comment]
Hey, could you provide more info about the program you're using?

**Reply by aidantorrence** [reply]
Google Chrome

**Comment by radar-blip** [comment]
You can file a bug report if you want, here: [Bug-Report](https://support.github.com/contact/bug-report)

**Comment by valen22br** [comment]
Yes, it is not working for me either. I'm using google chrome version 109.0.5414.119. 

I'll open a bug ticket.

**Reply by ghost** [reply]
Would you mind sharing the bug ticket link here, so we can vote and track it?
Thanks.

**Reply by valen22br** [reply]
Here is the one I opened:
https://support.github.com/ticket/personal/0/2000422

**Reply by msugakov** [reply]
The ticket appears to be personal therefore I can't see any progress there. Did you receive any updates on it @valen22br ?

**Reply by valen22br** [reply]
No news: 
![Screen Shot 2023-02-07 at 3 40 23 PM](https://user-images.githubusercontent.com/26287220/217360717-594ac6f4-05d6-4ce3-b4f5-33834bd87d3a.png)

**Comment by pedroazeredo04** [comment]
Yes for me neither, I'm on Firefox. I miss this tasty button very much

**Comment by milesfrain** [comment]
The "Expand all" button is broken for me on both chrome and firefox.

**Comment by etbkennedy** [comment]
Definitely broken, please fix...

**Comment by MOFFIO** [comment]
Fixed!

**Reply by pedroazeredo04** [reply]
Thanks !
